Spiral_Array/Spiral_Array_Python

```diff
@@ -0,0 +1,62 @@
+//Printing Spiral 2-D Matrix
+
```

Comment:
This file has no extension.
Remove language name from file name.

Reply:
I removed the language name from file. @ jainaman